x/gateway/keeper/msg_update_params.go

```diff
@@ -0,0 +1,26 @@
+package keeper
+
+import (
+	"context"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/pokt-network/poktroll/x/gateway/types"
+)
+
+func (k msgServer) UpdateParams(
+	goCtx context.Context,
+	req *types.MsgUpdateParams,
+) (*types.MsgUpdateParamsResponse, error) {
+	if k.GetAuthority() != req.Authority {
+		return nil, errorsmod.Wrapf(types.ErrGatewayInvalidSigner, "invalid authority; expected %s, got %s", k.GetAuthority(), req.Authority)
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	if err := k.SetParams(ctx, req.Params); err != nil {
```

Comment:
Did we get to the bottom of whether we will have to specify all params with each update? If that's still a constraint, would this be a good place to add a comment to track it?